ocean-client/addons/command-center/src/commands/help.ts

```diff
@@ -0,0 +1,46 @@
+import { ChangeTokenTo } from "./change-token-to";
+import { CheckMaxi } from "./check-maxi";
+import { Commands } from "./command";
+import { Execute } from "./execute";
+import { RemoveExposure } from "./remove-exposure";
+import { SetRange } from "./set-range";
+import { SetReinvest } from "./set-reinvest";
+import { SetToken } from "./set-token";
+import { Skip } from "./skip";
+import { StoreCommand } from "./store-command";
+
+// Krysh: not really nice, but making help command a store command to access our store
+export class Help extends StoreCommand {
```

Comment:
why store? does help need the store?

Reply:
help directly not really, but help does initialize all other commands and a few of them need a store to be initialized. We could make those stores optional to change it back to a normal Command. Would be fine for me too.

Reply:
such things should ring my bell, that something is a little bit off. Thanks for pointing out.
Both name() and description() were not right. Name is anyway defined inside Commands enum and description can be a static text on a Command itself.

Therefore changed from StoreCommand to Command as it is supposed to